pkg/lib/verifier/rules/pcr_eventlog_integrity.go

```diff
@@ -15,70 +17,126 @@ import (
 
 // NewPcrEventLogIntegrity creates a rule that will check if a PCR (in the host-manifest only)
 // has a "calculated hash" (i.e. from event log replay) that matches its actual hash.
-func NewPcrEventLogIntegrity(expectedPcr *types.Pcr, marker common.FlavorPart) (Rule, error) {
-	if expectedPcr == nil {
+func NewPcrEventLogIntegrity(expectedPcrLog *types.PCRS, expectedPcr *types.Pcr, marker common.FlavorPart) (Rule, error) {
```

Comment:
Again, why two parameters for expected PCRs?

Reply:
Here the two expected PCRS parameters are used for both the Linux and exsi flows as now both are having different structures (old and new implementation). When we start implementing dynamic verification for exsi flow, we can keep it as a single expected entry as these flows will have the same structure.